### PR TITLE
Added: dependabot.yml

### PR DESCRIPTION
**Pull request by GoliathLabs**
The dependabot.yml file has been added so that we no longer have to worry about dependency updates, since dependabot checks for them monthly. However, for this we should enable issues.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    directory: /
+    schedule: {interval: monthly}
+    reviewers: [@freifunkMUC/salt-stack]
+    assignees: [@freifunkMUC/salt-stack]
+
+  - package-ecosystem: npm
+    directory: /website
+    schedule: {interval: monthly}
+    reviewers: [@freifunkMUC/salt-stack]
+    assignees: [@freifunkMUC/salt-stack]
```
Comment (DasSkelett) [inline .github/dependabot.yml:28]: Can you put an empty line at the end? Makes it a bit tidier

**Review by DasSkelett — APPROVED** [review]
Looks good to me, maybe put a new line at the end, but that's not a blocker. I'm not in the salt-stack team so I won't receive the review pings, but I've got notifications turned on for this repository anyway.

File: .github/dependabot.yml (hunk 1 above)
Comment (GoliathLabs) [reply .github/dependabot.yml]: Done :)

**Comment by DasSkelett** [comment]
https://github.com/freifunkMUC/wg-access-server/runs/4071977549

> Dependabot couldn't parse the config file at .github/dependabot.yml. The error raised was:
> `(<unknown>): found character that cannot start any token while scanning for the next token at line 9 column 17`

**Comment by DasSkelett** [comment]
Initial wave of pull requests arrived:
- #15
- #16 
- #17
- #18
- #19
- #20

But no reviews requested or assignees set :thinking:

**Comment by GoliathLabs** [comment]
Hmm. Interesting. Maybe we should add you as the reviewer and assignee

**Pull request merged**